pymemcache/client/base.py

```diff
@@ -40,25 +40,41 @@
 
 # Some of the values returned by the "stats" command
 # need mapping into native Python types
+def _parse_bool_int(value):
+    return int(value) != 0
+
+
+def _parse_bool_string(value):
```

Comment:
Can we name this something slightly more descriptive, like `_parse_bool_string_is_yes` ?